docs-v2/reference/sdks/node.mdx

```diff
@@ -487,6 +531,50 @@ await nango.updateMetadata('<INTEGRATION-ID>', 'CONNECTION-ID', { 'CUSTOM_KEY1':
 **Response**
 
 Empty response.
+<Expandable>
+```json
+{
+    "CUSTOM_KEY1": "CUSTOM_VALUE1"
+}
+```
+</Expandable>
+
+### Bulk edit connection metadata
+
+Edit custom metadata for the connection. Only overrides specified properties, not the entire metadata.
+
+```js
+await nango.updateMetadata('<INTEGRATION-ID>', ['CONNECTION-ID-1', 'CONNECTION-ID-2'], { 'CUSTOM_KEY1': 'CUSTOM_VALUE1' });
+```
+
+**Parameters**
+
+<Expandable>
+    <ResponseField name="providerConfigKey" type="string" required>
+        The integration ID of the connection.
+    </ResponseField>
+
+    <ResponseField name="connectionIds" type="string[]" required>
+        The connection IDs.
+    </ResponseField>
+
+    <ResponseField name="metadata" type="Record<string, any>" required>
+        The custom metadata to store in the connection.
+    </ResponseField>
+</Expandable>
+
+**Response**
+
+<Expandable>
+```json
+{
+    "connection_ids": "<string[]>",
```

Comment:
I feel like it's clearer like, wdyt?
```suggestion
    "connection_ids": ["<string>"],
```

Reply:
Yeah, updated with https://github.com/NangoHQ/nango/pull/2145/commits/9d3557546cfc71fb5dfd2782a2decca09a5105b7